e2e-tests/action.yml

```diff
@@ -0,0 +1,168 @@
+name: e2e-tests
```

Comment:
This is the one we don't use in the end right?

Reply:
removed